### DISCUSSION TITLE
Sigil Support?

### DISCUSSION THREAD

**Discussion by cvkmohan** [mhanberg/temple#160] [Q&A] [answered]
@mhanberg - Thanks for the wonderful library. I slowly got hang of what temple does - and - how it fits into the complete cycle.
For each small component, we need to have two files - one ```*.ex``` and ```*.lexs```. Instead, like ```~H``` sigil for ```heex``` or ```~F``` sigil for surface components - will it improve Developer Experience, if there is a sigil support for temple template?

**Comment by mhanberg** [comment, marked as answer]
Please see this example for writing your markup inside your live view module. https://github.com/mhanberg/temple/discussions/111

Since temple is just elixir, there is no need for a sigil.

**Reply by cvkmohan** [reply]
Thanks! Sorry, if the questions were outright silly.
